src/backend/main/py.main/caliopen_main/user/store/device.py

```diff
@@ -8,16 +8,25 @@
 
 from datetime import datetime
 
-from caliopen_storage.store.model import BaseModel
+from caliopen_storage.store.model import BaseModel, BaseUserType
 
 from cassandra.cqlengine import columns
 
 log = logging.getLogger(__name__)
 
 
-class Device(BaseModel):
+class DeviceLocation(BaseUserType):
+    """Device known location, base on IP."""
+
+    location_ip = columns.Text(primary_key=True)    # IP address with CIDR
+    location_type = columns.Text()                  # home/work/etc
```

Comment:
if it can be edited by the user I would say it's a `location_name`. What do you think ?